src/Runner.Worker/FileCommandManager.cs

```diff
@@ -182,35 +185,69 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
                     return;
                 }
 
-                if (fileSize > AttachmentSizeLimit)
-                {
-                    context.Error(String.Format(Constants.Runner.UnsupportedSummarySize, AttachmentSizeLimit / 1024, fileSize / 1024));
-                    Trace.Info($"Step Summary file ({filePath}) is too large ({fileSize} bytes); skipping attachment upload");
+                var scrubbedFilePath = filePath + "-scrubbed";
+                
+                Trace.Verbose($"Step Summary file exists: {filePath} and has a file size of {fileSize} bytes");
+                // TODO: Figure out where to get the results service URL and check if it's set
```

Comment:
You have `IExecutionContext context` which should give you anything you want to for checking...